Rakefile

```diff
@@ -157,17 +98,40 @@ ENGINES = {
   :ruby_sass_3_2 => '3_2',
   :ruby_sass_3_3 => '3_3',
   :ruby_sass_3_4 => '3_4',
-  :libsass => 'lib',
+  :libsass_3_1 => 'libsass_3_1',
+  :libsass_3_2 => 'libsass_3_2',
+}
+
+#
+# Supported engines.
+#
+DOCKER_ENGINES = {
+  :ruby_sass_3_2 => 'xzyfer/docker-ruby-sass:3.2',
+  :ruby_sass_3_3 => 'xzyfer/docker-ruby-sass:3.3',
+  :ruby_sass_3_4 => 'xzyfer/docker-ruby-sass:3.4',
+  :libsass_3_1 => 'xzyfer/docker-libsass:3.1.0',
+  :libsass_3_2 => 'xzyfer/docker-libsass:3.2.0-beta.5',
 }
 
 #
-# Mapping with SassMeister endpoints.
+# Engine executable.
 #
-SM_ENGINES = {
-  '3_2' => '3.2',
-  '3_3' => '3.3',
-  '3_4' => '3.4',
-  'lib' => 'lib',
+ENGINE_EXEC = {
+  :ruby_sass_3_2 => nil,
+  :ruby_sass_3_3 => nil,
+  :ruby_sass_3_4 => nil,
+  :libsass_3_1 => nil,
+  :libsass_3_2 => nil,
+}
+
+#
+# Init each engine.
+#
+DOCKER_ENGINES.each { |engine, release|
+  unless ENGINE_EXEC[engine]
+    prefix = `boot2docker shellinit`.split(' ').values_at(1, 3, 5).join(' ')
```

Comment:
Docker needs some environment variables to function correct. These variables are exposed via `boot2docker shellinit`.

In order to avoid polluting the users environment I opted for prefixing all docker commands with the environment  variables so they're only present for that process.

Here I'm executing `boot2docker shellinit`, parsing the output, and building the prefix string.

```
$ boot2docker shellinit
Writing /Users/michael/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/michael/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/michael/.boot2docker/certs/boot2docker-vm/key.pem
    export DOCKER_CERT_PATH=/Users/michael/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1
    export DOCKER_HOST=tcp://192.168.59.103:2376
```

becomes 

```
prefix = "DOCKER_HOST=tcp://192.168.59.103:2376 DOCKER_CERT_PATH=/Users/michael/.boot2docker/certs/boot2docker-vm DOCKER_TLS_VERIFY=1"
```

This value unfortunately cannot be hardcoded as it's dependant on factors we can't know.